### PR TITLE
pdfjoin: add page

### PR DESCRIPTION
**Pull request by dcervenkov**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/pdfjoin.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# pdfjoin
+
+> PDF merging utility.
```
Comment (pxgamer) [inline pages/common/pdfjoin.md]: Is this `pdfjoin` part of the [`pdfjam`] package by David Firth? Or another package?

[`pdfjam`]: http://go.warwick.ac.uk/pdfjam

File: pages/common/pdfjoin.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- Merge two PDFs:
+
+`pdfjoin {{file1}} {{file2}} {{output_file}}`
```
Comment (pxgamer) [inline pages/common/pdfjoin.md]: If this is the `pdfjam` component, I think this needs to use `--outfile {{path/to/output}}`?

Comment (dcervenkov) [reply pages/common/pdfjoin.md]: Oops, you're right, I missed the switch. It indeed is part of `pdfjam`.

File: pages/common/pdfjoin.md (hunk 1 above)
Comment (sbrl) [reply pages/common/pdfjoin.md]: Looks like it is part of `pdfjam`:

```bash
$ pdfjoin --help | head

pdfjam is a shell-script front end to the LaTeX 'pdfpages' package (for
which, see http://www.ctan.org/tex-archive/macros/latex/contrib/pdfpages).

Usage: pdfjam [OPTIONS] [--] [FILE1 [SEL1]] [FILE2 [SEL2]]...
where.....
```

**Comment by pxgamer** [comment]
@sbrl @agnivade, I assume this is ok to be merged?

**Comment by agnivade** [comment]
Yes

**Pull request merged**